app/site/_includes/nav.liquid

```diff
@@ -1,39 +1,63 @@
 <div class="usa-overlay"></div>
 <header class="usa-header usa-header--basic">
-  <div class="usa-nav-container">
-    <div class="usa-navbar">
+  <div class="usa-navigation-container">
+    <div class="usa-nav-bar">
+      <div class="title">
+        <img src="{{ "/assets/img/cms-logo.jpg" | url }}" class="logo">
+        <h1>Metrics Website</h1>
+      </div>
       <div class="usa-logo">
         <em class="usa-logo__text">
           <a href="{{ "/" | url}}" title="{{site.title}}">{{ site.title }}</a>
         </em>
       </div>
-      <button type="button" class="usa-menu-btn">Menu</button>
     </div>
     <nav aria-label="Primary navigation" class="usa-nav">
       <button type="button" class="usa-nav__close">
-        <img src="{{"/assets/img/usa-icons/close.svg" | url}}" role="img" alt="Close">
+        <svg class="usa-icon--size-7" aria-hidden="true" focusable="false" role="img">
+          <use xlink:href="/assets/img/sprite.svg#close"></use>
```

Comment:
This icon is not showing up when I run `npm run prod`. This is not being able to find the path to this image since due to prod prepending "/metrics" to every path. To fix, I think the 11ty url filter needs to be used:
`{{"/assets/img/usa-icons/close.svg" | url }}`
https://www.11ty.dev/docs/filters/url/
<img width="1579" alt="Screenshot 2024-12-02 at 10 18 30 AM" src="https://github.com/user-attachments/assets/b8047033-c830-40a1-bd8b-1ca902edbf3e">
(Blue line is the menu icon)